test/integration/common

```diff
@@ -83,6 +83,25 @@ check-synced-entry() {
   fail-now "timed out waiting for agent to sync down entry"
 }
 
+check-svid-count() {
+  MAXCHECKS=50
+  CHECKINTERVAL=1
+
+  for ((i=0;i<=MAXCHECKS;i++)); do
+    if (( $i>=$MAXCHECKS )); then
+      fail-now "svid count validation failed"
+    fi
+    log-info "check svid count on agent debug endpoint ($(($i+1)) of $MAXCHECKS max)..."
+    COUNT=`docker-compose exec -T $1 /opt/spire/conf/agent/debugclient -testCase "printDebugPage" | jq '."svidsCount"'`
```

Comment:
```suggestion
    COUNT=`docker-compose exec -T $1 /opt/spire/conf/agent/debugclient -testCase "printDebugPage" | jq '.svidsCount'`
```
The double quotes are not necessary in the `jq` command

Reply:
FYI, I filed https://github.com/spiffe/spire/issues/3256, as it is currently ambiguous what the `svidsCount` field should represent. Since today this value is the count of the number of X.509-SVIDs in the primary cache, I would propose we make this distinction clear in the test by using the term `X.509-SVID` rather than the more generic term `SVID` in all the respective variable names and log messages in this test.

Reply:
sounds good!